scixblog/_posts/2024-02-26-userfeedback-2024.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: blog_post_scix
+title: "Informed Transformation: User Feedback’s Role in the ADS to SciX Transition"
+author: Brit Myers
+position: SciX Community Engagement Consultant
+category: scixblog
+label: news
+thumbnail: blog_2024-02-26_surveyresponse_1.png
```

Comment:
Missing the folder - should be `blog/images/blog_2024-02-26_surveyresponse_1.png`

Reply:
updated